projects/kit/components/input-slider/input-slider.template.html

```diff
@@ -1,78 +1,38 @@
-<tui-wrapper
-    [appearance]="appearance"
+<tui-input-number
+    [min]="min"
+    [max]="max"
+    [precision]="precision"
+    [decimal]="decimal"
+    [prefix]="prefix"
+    [postfix]="postfix || (value | i18nPlural: pluralizeMap || pluralizeMapFallback)"
+    [tuiTextfieldCustomContent]="controller.customContent || deprecatedSecondary"
+    [tuiTextfieldSize]="computedSize"
+    [tuiTextfieldLabelOutside]="computedSize !== 'l'"
     [readOnly]="readOnly"
-    [disabled]="disabled"
-    [focused]="computedFocused"
-    [hovered]="computedHovered"
-    [invalid]="computedInvalid"
+    [pseudoFocused]="computedFocused"
+    [pseudoHovered]="pseudoHovered"
```

Comment:
Please use `computed` values for now, they are deprecated but still needed until 3.0 to preserve possible custom appearances. Let's just replace them all together later.

Reply:
fixed for `InputSlider` and for `InputNumber`